src/pfe/file-watcher/server/test/cronjob.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env bash
+
+# Colors for success and error messages
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+BLUE='\033[0;36m'
+RESET='\033[0m'
+
+# Set up variables
+CW_DIR=~/codewind
+CW_TEST_DIR=$CW_DIR/src/pfe/file-watcher/server/test
+CODEWIND_REPO=git@github.com:eclipse/codewind.git
+
+# Make all options mandatory to ensure cronjob admin knows what exectly each cronjob does, it's for debug purpose
+function usage {
+    me=$(basename $0)
+    cat <<EOF
+Usage: $me: [-<option letter> <option value> | -h]
+Options:
+    -t # Test type, currently support 'local' and 'kube' - Mandatory
+    -s # Test suite - Mandatory
```

Comment:
Did we discuss to make this optional and fallback to functional for default?

Reply:
See the command above `# Make all options mandatory to ensure cronjob admin knows what exectly each cronjob does, it's for debug purpose`

Reply:
I am thinking of user only uses `tesh.sh`, we make all options optional and set up default options value for users' convenience. But `cronjob.sh` is only for our cronjob/nightly run, it's better to force us to fill out all options for debug purpose

Reply:
Ok. Makes sense.